### DISCUSSION TITLE
How to disable some feature : utf8

### DISCUSSION THREAD

**Discussion by VoidScute** [wezterm/wezterm#5410] [General]
Hello,

I currently can't edit my bash script because != are always replace by the ≠ symbol.
How can i disable that ? 

thx

**Comment by wez** [comment]
Those are *ligatures*. See https://wezfurlong.org/wezterm/config/font-shaping.html for information on configuring (including disabling) the font shaper that performs those visual substitutions.

**Comment by VoidScute** [comment]
thx a lot, i was stuck on that since days :D
